blocks/init/src/Blocks/components/modal/modal.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * Template for the Modal Component.
+ *
+ * @package EightshiftBoilerplate.
+ */
+
+use EightshiftBoilerplateVendor\EightshiftLibs\Helpers\Components;
+
+$globalManifest = Components::getManifest(dirname(__DIR__, 2));
+$manifest = Components::getManifest(__DIR__);
+
+$modalUse = Components::checkAttr('modalUse', $attributes, $manifest);
+
+if (!$modalUse) {
+	return;
+}
+
+$unique = Components::getUnique();
+
+$componentClass = $manifest['componentClass'] ?? '';
+$additionalClass = $attributes['additionalClass'] ?? '';
+$blockClass = $attributes['blockClass'] ?? '';
+$selectorClass = $attributes['selectorClass'] ?? $componentClass;
+$componentJsClass = $manifest['componentJsClass'] ?? '';
+
+$modalExitButton = Components::checkAttr('modalExitButton', $attributes, $manifest);
+$modalContent = Components::checkAttr('modalContent', $attributes, $manifest);
+$modalId = Components::checkAttr('modalId', $attributes, $manifest);
+
+$modalClass = Components::classnames([
+	Components::selector($componentClass, $componentClass),
+	Components::selector($blockClass, $blockClass, $selectorClass),
+	Components::selector($additionalClass, $additionalClass),
+	Components::selector($componentJsClass, $componentJsClass),
+]);
+
+$modalOverlayClass = Components::selector($componentClass, $componentClass, 'overlay');
+$modalDialogClass = Components::selector($componentClass, $componentClass, 'dialog');
+$modalContentClass = Components::selector($componentClass, $componentClass, 'content');
+$modalExitButtonClass = Components::selector($componentClass, $componentClass, 'close-button');
+
+?>
+
+<div
+	class="<?php echo \esc_attr($modalClass); ?>"
+	data-id="<?php echo \esc_attr($unique); ?>"
+	id="<?php echo \esc_attr($modalId); ?>"
+	aria-hidden="true"
+>
+	<?php
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
ignores goes in the same line as the function after. please check the other components

Reply:
Ignore statements count towards line length limits, so some components' PHP implementations (image, column, columns, ...) put them on the line above to avoid having another lint error. Seems okay to me :shrug:
(It applies to the next line if there isn't PHP code on the same line)

Reply:
i generally don't like for comments to interfere in reading the code but it is ok

Reply:
it's not problem, I'll fix it 😃